configs/setr/README.md

```diff
@@ -36,8 +36,52 @@ This head has two version head.
 }
 ```
 
+
+## Usage
+
+To use the pre-trained ViT model, you need to convert its keys.
+
+We provide a script [`vitjax2mmseg.py`](../../tools/model_converters/vitjax2mmseg.py) in the tools directory to convert the keys of models from [ViT-AugReg](https://github.com/rwightman/pytorch-image-models/blob/f55c22bebf9d8afc449d317a723231ef72e0d662/timm/models/vision_transformer.py#L54-L106) to MMSegmentation style.
+
```

Comment:
Is there any experimental result with jax pre-train?

Reply:
Nope